### ISSUE TITLE
Characters like ^M are almost invisible

### ISSUE THREAD

**Issue by nickknw** [altercation/vim-colors-solarized#5]
It looks like they are this colour so that it is easier to filter them out, but the shade they are at is so close to the background colour (with both light and dark backgrounds) that they are incredibly hard to distinguish. 

Another good example on the vim welcoming screen, where the `<Enter>` and `<F1>` bits look like they are absent.

[Screenshot of what I'm talking about.](http://i.imgur.com/53Kca.png)

**Comment by nickknw** [comment]
I realize this will probably differ a bit depending on the monitor and monitor settings you have. I've checked on both of my home monitors and both of my work monitors (4 different models, all have different settings) and it is roughly equally difficult on all of them.

**Comment by blueyed** [comment]
The same applies to the chars being displayed with `set list` (it may be the same color group).
I would like those to be more visible, too. (I have a mapping to toggle "list", and if it's enabled I do not want to see it for the current line only).

**Comment by ask** [comment]
@blueyed++

Love the theme, but ':set list' chars is almost invisible, it made me commit lots of extraneous whitespace yesterday :(

**Comment by altercation** [comment]
I'll add an option to make them more visible.

**Comment by altercation** [comment]
Addressed in [commit 9f2b8bc](https://github.com/altercation/vim-colors-solarized/commit/9f2b8bc69ddd589156d92fbdc9800c0e43328368).

I've added a new setting to allow for different non-text and special character highlighting levels.

```
------------------------------------------------
g:solarized_visibility =  "normal" (default)   "high" or "low"
------------------------------------------------
Special characters such as trailing whitespace, tabs, newlines, when 
displayed using ":set list" can be set to one of three levels depending on 
your needs.
```

See the README for more information.
